src/lib/migration/proposal_store.rb

```diff
@@ -0,0 +1,70 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2014 Novell, Inc. All Rights Reserved.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may find
+# current contact information at www.novell.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "installation/proposal_store"
+
+module Migration
+  # Provides access to static metadata for update proposal.
+  class ProposalStore < Installation::ProposalStore
+    include Yast::Logger
+    include Yast::I18n
+
+    PROPOSAL_NAMES = ["update_proposal"]
+    PROPOSAL_PROPERTIES = {}
+    MODULES_ORDER = PROPOSAL_NAMES
+
```

Comment:
missing textdomain for translation. Also it have to have initialization with argument due to way how it is initialized so something like:
```
def initialize(_unused_mode)
  textdomain "migration"
 
  super("migration")
end
```